include/can/core/message_handlers/motor.hpp

```diff
@@ -48,6 +48,14 @@ class MotorHandler {
         motor.motion_controller.move(mv_msg);
     }
 
+    void visit(EnableMotorRequest &m) {
```

Comment:
in the python can_comms script is this the setup_request message?

Reply:
No, I have to add new messages, which I did [here](https://github.com/Opentrons/opentrons/pull/8531).